crates/types/src/consensus.rs

```diff
@@ -732,7 +733,9 @@ pub struct CommitmentAndMetadata<TYPES: NodeType> {
     /// Metadata for the block payload
     pub metadata: <TYPES::BlockPayload as BlockPayload<TYPES>>::Metadata,
     /// Builder fee data
-    pub fee: BuilderFee<TYPES>,
+    pub fees: Vec1<BuilderFee<TYPES>>,
```

Comment:
For some reason I thought we were using a `Vec1` for this in some other place, but let me look at it again!

Reply:
I think the rationale in my original design was that, since `BuilderFee` was not an `Option` type, then we needed always put *something* in the `fee` field. Therefore, it seemed sensible to use `vec1` for backwards compatibility to guarantee that the existing behavior stayed.

Reply:
I think we just put in `null_block::builder_fee()` currently, if we receive no fees

Reply:
Okay, then I suppose we can do the same for the marketplace.